### DISCUSSION TITLE
Is it possible to disable "Require branches to be up to date before merging"?

### DISCUSSION THREAD

**Discussion by j0k3r** [serverless-heaven/serverless-webpack#1553] [Team Posts]
I found it a bit boring to always ensure a PR is up to date with the master before being able to merge it.

![image](https://user-images.githubusercontent.com/62333/92212138-cc17c580-ee91-11ea-97b4-f642190ffe09.png)

FMPOV, the only fact the PR hasn't conflict is enough. And if now something will broke, we should manually update the without being forced to do so.

What do you thin @HyperBrain?

**Comment by HyperBrain** [comment]
The reason for this is more, that the tests are run on the branch in a
"like merged" state.
Any quality assurance (e.g. unit tests) done on a branch that is not up to
date with master does not guarantee anything (i.e. if you have a change
based on a months old branch - without conflicts - this does not tell you
that it will semantically integrate correctly)

**Comment by j0k3r** [comment]
All right, I'll keep rebasing then
